openquake/mbt/tools/faults.py

```diff
@@ -92,99 +122,108 @@ def rates_for_double_truncated_mfd(area, slip_rate, m_low, m_upp, b_gr,
     :parameter slip_rate:
         Slip-rate
         float [mm/tr]
-    :parameter m_low:
-        Lower magnitude
+    :parameter m_min:
+        Minimum magnitude
         float
-    :parameter m_upp:
-        Upper magnitude
+    :parameter m_max:
+        Maximum magnitude
+        float
+    :parameter m_cli:
+         Clipping magnitude
+         float
     :parameter b_gr:
         b-value of Gutenber-Richter relationship
     :parameter bin_width:
         Bin width
     :parameter rigidity:
         Rigidity [Pa]
     :return:
-        A list containing the rates per bin starting from m_low
+        A list containing the rates per bin starting from m_min
+        A list containing the magnitude bins starting from m_min
```

Comment:
The order of this in inverted i.e. first you return magnitudes than rates. For rates it would be better to say IMO: `A list containing values of magnitude located at the centre of each magnitude bin, starting from mmin+bin_width/2`

Reply:
lgtm [order]
rates: A list containing values of rate located at the centre of each magnitude bin, starting from mmin+bin_width/2